docs/src/modules/brandingTheme.ts

```diff
@@ -110,7 +110,7 @@ export const getDesignTokens = (mode: 'light' | 'dark') =>
   ({
     palette: {
       primary: blue,
-      divider: mode === 'dark' ? blueDark[400] : grey[200],
+      divider: mode === 'dark' ? blueDark[700] : grey[200],
```

Comment:
![image](https://user-images.githubusercontent.com/18292247/130598850-95e70ca6-1969-4823-8073-790dc8d090bd.png)

```suggestion
      divider: mode === 'dark' ? blueDark[500] : grey[200],
```
I think it should be lighter than `700`. So far, `700, 800, 900` are background.